src/@types/status.ts

```diff
@@ -10,6 +10,11 @@ export interface Status {
    * @defaultValue 0
    */
   toxicTurnCount: number;
+  /**
+   * The number of turns the affected Pokemon has slept since
```

Comment:
```suggestion
   * The number of turns the affected Pokemon has been sleeping
```

Should we specify at which point this number is incremented? I assume at the end of each sleeping turn?

Reply:
Probably not necessary since it's updated at the same time as the other fields here